### DISCUSSION TITLE
Adding Harmony one Shard1

### DISCUSSION THREAD

**Discussion by marsidorowicz** [wevm/wagmi#1831] [Ideas]
Is your feature request related to a problem? Please describe.
Adding Harmony one Shard1

Describe the solution you'd like
Add chain harmony as follows or similar:

```
export const harmony: Chain = {
	id: 1666600000,
	name: 'Harmony One',
	network: 'harmony',
	nativeCurrency: {
		decimals: 18,
		name: 'Harmony',
		symbol: 'ONE'
	},
	rpcUrls: {
		public: { http: ['https://rpc.ankr.com/harmony'] },
		default: { http: ['https://rpc.ankr.com/harmony'] }
	},
	blockExplorers: {
		etherscan: { name: 'Ankr', url: 'https://rpc.ankr.com/harmony' },
		default: { name: 'Ankr', url: 'https://rpc.ankr.com/harmony' }
	},
	contracts: {
		multicall3: {
			address: '0xca11bde05977b3631167028862be2a173976ca11',
			blockCreated: ??? needs blockCreated !!!!!!!!!!!
		}
	}
}
```

**Comment by tmm** [comment]
Feel free to open a PR in the [references](https://github.com/wagmi-dev/references) repository.

**Comment by marsidorowicz** [comment]
can you explain a bit how? I'm new in this :)

**Reply by tmm** [reply]
1. Clone the references repo
2. Add a `harmony.ts` file to `packages/chains/src/`
3. Add the Chain to this file (make sure you add all the [required properties](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#chains))
4. Export the Chain from `packages/chains/src/index.ts`
5. Make sure tests pass (update snapshot for `index.test.ts`)
6. Create changeset

Check out the [Contributing Guide](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) for more info.

**Comment by marsidorowicz** [comment]
It is on PR https://github.com/wagmi-dev/references/pull/121 One test didn't pass

![image](https://user-images.githubusercontent.com/36972693/219878111-f61bb980-6eb4-49d8-9c63-4350788c80bc.png)

do I need to provide my projectId for testing?

**Comment by marsidorowicz** [comment]
that was it:

![image](https://user-images.githubusercontent.com/36972693/219883438-4bcea4bf-f182-474b-8522-2df5cad7d013.png)
